pay-api/tests/unit/api/test_account.py

```diff
@@ -475,6 +475,39 @@ def test_update_pad_account_when_cfs_up(session, client, jwt, app):
     assert rv.status_code == 200
 
 
+def test_switch_eft_account_when_cfs_down(session, client, jwt, app, admin_users_mock):
+    """Assert that the payment records are created with 202."""
+    token = jwt.create_jwt(get_claims(role=Role.SYSTEM.value), token_header)
+    headers = {'Authorization': f'Bearer {token}', 'content-type': 'application/json'}
+    rv = client.post('/api/v1/accounts',
+                     data=json.dumps(get_basic_account_payload(payment_method=PaymentMethod.PAD.value)),
+                     headers=headers)
+    auth_account_id = rv.json.get('accountId')
+    # Mock ServiceUnavailableException
```

Comment:
redundant - could remove or include more details that the code below doesn't include